metals/src/main/scala/scala/meta/internal/metals/debug/DebugProvider.scala

```diff
@@ -497,7 +497,9 @@ class DebugProvider(
     val socket = new Socket()
 
     val address = new InetSocketAddress(uri.getHost, uri.getPort)
-    val timeout = TimeUnit.SECONDS.toMillis(10).toInt
+    def isCI: Boolean = "true" == System.getenv("CI")
+    val timeoutDuration = if (isCI) 60 else 10
```

Comment:
Even 60 seconds do not seem enough :/ the java process doesn't seem to be able to start, or is it jsut sockets?